### DISCUSSION TITLE
Adding new store do not show up

### DISCUSSION THREAD

**Discussion by Pluegpu** [jef/streetmerchant#2508] [Help]
I copied an existing store, added to another country-ending, changed store name
I added a new store where the other stores are and also added the store to the index-file.

There are no big letters in the name. It is an exact copy of another store, only different country.

Is there a file where you add a new country? Also currency maybe?

It still does shows "store does not exist"

**Comment by Aragorn450** [comment]
See my answer in https://github.com/jef/streetmerchant/discussions/2511 for this answer. It should be the same. If you still can't get it to work, please attach the new store file and the index.ts file you modified so we can take a look and see what is incorrect.
